admin/build_dev_wheels.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+REPO_ROOT="$(realpath "$(dirname $0)/..")"
+
+# if you edit this or specify a new value make sure the packages are in dependency order!
+REPOS=${REPOS-archinfo pyvex claripy cle ailment angr}
+
+OUTDIR=${OUTDIR-./build}
+mkdir -p "$OUTDIR"
+
+python -m pip install -U setuptools wheel build
```

Comment:
If this is an already-installed angr-dev environment, setuptools and wheel aren't needed since their dependencies would be asserted previously.

Reply:
This is more for the upgrade command than the install command. I had a pretty old version of setuptools in my angr env and I imagine others do too.